### DISCUSSION TITLE
When zooming out the screen, the divider in the table header occasionally disappears

### DISCUSSION THREAD

**Discussion by Zoey-Guo-RSP** [ant-design/ant-design#51164] [Q&A]
![tableHeader-Divider](https://github.com/user-attachments/assets/c9638b2d-cd92-4209-b68a-af91d71aeedf)

As shown in the figure, when the screen is reduced to 90%, some of the table header boundaries will disappear.
This is done on the antd website for the sample table of version 4.24.16. And I've noticed that this issue also appears in the sample table of 5.21.2.

Is there a way to fix this?

**Comment by zombieJ** [comment]
Use a higher resolution screen. It sounds funny, but it can indeed solve some problems. Under scaling, pixels can also be clipped. Whether 1px can be displayed depends on the browser's implementation logic and the resolution of the external device.

**Reply by Zoey-Guo-RSP** [reply]
Could you tell me the specific screen resolution? Or is there another solution？

**Comment by eryet** [comment]
Looking for solution as well, customer were nitpicking about this 😭
